packages/ember-data/lib/system/store.js

```diff
@@ -2135,13 +2108,16 @@ function setupRelationships(store, record, data) {
       relationship = record._relationships.get(key);
       relationship.updateMeta(data.relationships[key].meta);
     }
+
     var value = data.relationships[key].data;
```

Comment:
Mostly for code readability/place to put the deleted comment/ease of
understanding
On Oct 22, 2015 10:04, "Trent Willis" <notifications@github.com> wrote:

> In packages/ember-data/lib/system/store.js
> <https://github.com/emberjs/data/pull/3854#discussion_r42775635>:
>
> > @@ -2135,13 +2108,16 @@ function setupRelationships(store, record, data) {
> >        relationship = record._relationships.get(key);
> >        relationship.updateMeta(data.relationships[key].meta);
> >      }
> > +
> >      var value = data.relationships[key].data;
>
> I can, but not sure I see the point in doing that. If we normalize here,
> then we will have to check that value exists and the kind of relationship
> in two places (here and then again below).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/emberjs/data/pull/3854/files#r42775635>.
>